tests/unit/parsers/test.manifestjson.js

```diff
@@ -213,7 +213,7 @@ describe('ManifestJSONParser', () => {
       );
       expect(manifestJSONParser.isValid).toEqual(true);
       const metadata = manifestJSONParser.getMetadata();
-      expect(metadata.manifestVersion).toEqual(VALID_MANIFEST_VERSION);
+      expect(metadata.manifestVersion).toEqual(MANIFEST_VERSION_VALID);
```

Comment:
Ha. I see now, but I am still confused. The previous constant reads a bit better but I don't there is a single value that is valid these days (or soon). Shouldn't we have a list of valid manifest versions? And maybe a default one?